src/mercury/boot/views.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 Ruben Quinones (ruben.quinones@rackspace.com)
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import os
+import logging
+
+from flask import request, send_file, abort, jsonify
+from flask.views import MethodView
+
+from werkzeug.utils import secure_filename
+
+from mercury.common.clients.inventory import InventoryClient
+from mercury.boot.configuration import get_boot_configuration
+
+
+log = logging.getLogger(__name__)
+configuration = get_boot_configuration()
+
+
+class BootView(MethodView):
+    """
+    Boot method view
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(BootView, self).__init__(*args, **kwargs)
+        inventory_url = configuration.inventory.inventory_router
+        self.inventory_client = InventoryClient(inventory_url)
+
+    def get(self, mac_address=None):
+        """
+        Main boot endpoint 
+
+        :param mac_address: Inventory object mac_address, default is None.
+        :return: Boot file.
+        """
+        if mac_address:
+            result = self.inventory_client.query(
+                {
+                    'interfaces.address': mac_address
+                }
+            )
+
+            try:
+                inventory = result['message']['items'][0]
+            except IndexError:
+                return abort(404)
```

Comment:
Technically this can be commented out too for now right?

Reply:
@hussamd correct, I will comment it out but would like to confirm if we do need support for platform/device specific boot options or treat devices already registered in a different way.

Reply:
@hussamd Perhaps if we do ever need this it would be best to create some sort of pxe "rules" api, where you can add/remove/update boot options if a device in inventory matches certain criteria.